dpgen/dispatcher/LSF.py

```diff
@@ -102,14 +102,19 @@ def sub_script_head(self, res):
         ret = ''
         ret += "#!/bin/bash -l\n#BSUB -e %J.err\n#BSUB -o %J.out\n"
         if res['numb_gpu'] == 0:
-            ret += '#BSUB -R span[ptile=%d]\n#BSUB -n %d\n' % (
-                res['node_cpu'], res['numb_node'] * res['task_per_node'])
+            ret += '#BSUB -n %d\n#BSUB -R span[ptile=%d]\n' % (
```

Comment:
Why modify this line? It seems that it’s as same as before.

Reply:
Only for the same as the code below, without any other meaning.